packages/shared/src/utils/debug/dbPerfMonitor.ts

```
@@ -301,6 +305,15 @@
     }
     logResult({ isWarning: true });
 
+    if (
+      !isNil(generalDebuggerRule[key]) &&
+      indexedDBResult[key] >= generalDebuggerRule[key]
+    ) {
+      if (settings?.debuggerEnabled) {
+        debugger;
+      }
+    }
+
```

Biome (2.1.2)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)